src/books.css

```diff
@@ -7,8 +10,19 @@
   display: flex;
```

Comment:
- [ ] You have used `display: flex` 6 times and this way you are repeating the same rule which falls under the of bad practicing with CSS. Kindly create a class with this function and give to any HTML item you want

Reply:
Hey @GzimAsani, you're right. Thank you.